Bookish/Bookish.csproj

```diff
@@ -14,4 +14,8 @@
       <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.6" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Content Remove="Views\Home\EditMember.cshtml" />
+    </ItemGroup>

```

Comment:
What's this about?

Reply:
No idea, I think it was automatically generated